parlai/utils/world_logging.py

```diff
@@ -74,12 +75,17 @@ def _add_msgs(self, acts, idx=0):
         """
         msgs = []
         for act in acts:
+            # padding examples in the episode[0]
+            if isinstance(act, Message) and act.is_padding():
```

Comment:
only filter out if act is `Message` otherwise it'll break the unittests for `act` is `dict`